app/assets/stylesheets/variables.scss

```diff
@@ -28,11 +29,8 @@ $link-decoration: none; // no underline
 $navbar-dark-color: $white;
 
 // Accordion
-$accordion-button-color: $white;
-$accordion-button-bg: $sonic-silver;
-$accordion-button-active-color: $white;
-$accordion-button-active-bg: $sonic-silver;
-$accordion-button-focus-box-shadow: 0 0 0.25rem 0.25rem rgba($sonic-silver, 0.25);
+$accordion-button-bg: $black10;
```

Comment:
Consider using CSS properties `--bs-accordion-bg` and `--bs-accordion-active-bg` instead

Reply:
It doesn't look like BS 5.1 has CSS variables for accordions: https://getbootstrap.com/docs/5.1/components/accordion/#variables